### DISCUSSION TITLE
追求卓越（完美）一定对吗？ | /cn/2022-06-20-/

### DISCUSSION THREAD

**Discussion by giscus** [JinjiPang/jinji#51] [Comments]
# cn/2022-06-20-/

Follow your excellence, and success will chase you.
追求卓越，成功自然会追着你跑。 &mdash;《Three idiots》
[&hellip;] 我曾多次提到自己可能不适合做科研，这倒不是我觉得科研辛苦，而是我总是在怀疑自己的那些研究结果到底对不对。或者不仅仅是我自己的研究，我对发表了的很多研究也深表怀疑。
就拿我研究的微生物区来讲，研究两个 &hellip;

https://jinjipang.com/cn/2022-06-20-/

**Comment by liechi** [comment]
快一年过去了，不知对该问题有答案了吗？

**Reply by JinjiPang** [reply]
无解，找了个最新的方法（ANCOM-BC）写上去了，估计这周投出去。

**Reply by liechi** [reply]
祝好运。

**Reply by JinjiPang** [reply]
已发表。还是有不完美的地方。

**Reply by liechi** [reply]
祝贺。挺有意思的研究。

文章千古事，得失寸心知。有遗憾很正常，觉得自己的文章完美无缺反倒不一定是好事。

祝你接下来求职顺利 : )

**Reply by JinjiPang** [reply]
谢谢！
